pkg/install/gatherlogs.go

```diff
@@ -32,8 +36,16 @@ func (i *Installer) gatherFailureLogs(ctx context.Context) {
 			continue
 		}
 
-		i.log.Printf("%s: %s", runtime.FuncForPC(reflect.ValueOf(f).Pointer()).Name(), string(b))
+		i.log.Printf("%s: %s", log.FriendlyName(logFunc), string(b))
+	}
+
+	if _, ok := origErr.(*api.CloudError); !ok {
+		err = fmt.Errorf("%s: %s", f.Name(), origErr)
```

Comment:
what are you trying to do here? it's not always a good idea to wrap errors like this

Reply:
I'm not sure, this was in the original code: https://github.com/Azure/ARO-RP/blob/318bd55a156f117f3d61528fb30f6f9de1a0e11f/pkg/install/install.go#L272-L274